common/src/main/java/net/william278/husksync/data/DataServerID.java

```diff
@@ -0,0 +1,37 @@
+package net.william278.husksync.data;
+
+import com.google.gson.annotations.SerializedName;
+import org.jetbrains.annotations.NotNull;
+
+public class DataServerID {
+
+    @SerializedName("server_id")
+    @NotNull
+    private static String serverID;
+
+
+    /**
+     * Set the {@link DataServerID}
+     *
+     * @param server the {@link DataServerID} to set
+     * @return this {@link DataServerID#serverID}
+     * @since 2.2
+     */
+    @NotNull
+    public static String setServerID(@NotNull String server) {
```

Comment:
Not fond of static methods for stuff like this, would rather see this String be accessed through a getter in the HuskSync plugin interface directly